File: Service_eHospital_DongNai_A/Implement/HoSoSucKhoe_TiepXucService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class HoSoSucKhoe_TiepXucService : BaseService<HoSoSucKhoe_TiepXuc, IHoSoSucKhoe_TiepXucRepository>
    , IHoSoSucKhoe_TiepXucService
    {
    private readonly IHoSoSucKhoe_TiepXucRepository _HoSoSucKhoe_TiepXucRepository;
    public HoSoSucKhoe_TiepXucService(IHoSoSucKhoe_TiepXucRepository HoSoSucKhoe_TiepXucRepository) : base(HoSoSucKhoe_TiepXucRepository)
    {
    _HoSoSucKhoe_TiepXucRepository = HoSoSucKhoe_TiepXucRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/HoSoSucKhoe_TiepXuc.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { HoSoSucKhoe_TiepXuc } from './HoSoSucKhoe_TiepXuc.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class HoSoSucKhoe_TiepXucService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'TiepXuc_Id', 'HoSoSucKhoe_Id', 'TinhChat_Id'];

    List: HoSoSucKhoe_TiepXuc[] | undefined;
    ListFilter: HoSoSucKhoe_TiepXuc[] | undefined;
    FormData!: HoSoSucKhoe_TiepXuc;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "HoSoSucKhoe_TiepXuc";
    }
}

